README.md

```diff
@@ -27,3 +27,15 @@ In order to apply the headers, run:
  -$ ./gradlew license
  ```
 NOTE: The license check is executed as part of the standard `build` target.
+
+## Creating packages for Linux, Windows and Mac
+The gradle build includes a task which creates three packages by bundling OpenJDK, a fat JAR of this project and a system-specific launcher script.
+For Linux user, extract the package and run : ./Linux-launcher.sh
+For Windows user, extract the package and double click Windows-launcher.bat
+For Mac user, extract the package and (to be continued...)
```

Comment:
Keep in mind that white space/blank lines are important in Markdown. The above is all running together.

![Screenshot_2019-05-17 apereo openEQUELLA-admin-console-package](https://user-images.githubusercontent.com/43919233/57895080-23ed8000-788d-11e9-8ebc-196dbf23425b.png)


I suggest that you also make the list into an actual bulleted list, ie.

```
* For Linux ...
* For Windows ...
* For Mac ...
```